src/components/NcDateTimePicker/NcDateTimePicker.vue

```diff
@@ -477,10 +477,53 @@ export default {
 		}
 	}
 }
-
+.v-select.select {
+	min-width: 246px;
+	left: -8px !important;
+}
 // TODO: This should be scoped or targeted by a specific selector, but the NcSelect component does not allow this yet.
 .vs__dropdown-menu--floating {
 	// Higher z-index than the popover in which the NcSelect is located.
 	z-index: 100001 !important;
 }
+
+.mx-datepicker-main .mx-table .cell.disabled {
+	border-radius: var(--border-radius-small);
+	background-color: var(--color-background-dark);
+	color: var(--color-text-maxcontrast);
+	opacity: 1;
+}
+
+.mx-datepicker-main .mx-table .cell {
+	border-radius: var(--border-radius-small);
+	min-height: 30px !important;
```

Comment:
This should be probably either default clickable or clickable-small right?

Reply:
yes, and also, i need to fix the classes to be shown as BEM.

Reply:
But that are internal classes, so no renaming to BEM possible, no?

Reply:
i meant:

```
main {
.cell {
.today {
.other.. {
}
 }
  }
   }
```